src/rust/Cargo.toml

```diff
@@ -18,6 +18,7 @@ geozero = "0.5.1"
 async-trait = "0.1"
 reqwest = "0.10"
 bytes = "0.5"
+log = "0.4"
```

Comment:
Note this is already a transitive dependency. Without setting up a logging back-end nothing will actually be logged.